db/migrate/20160411150441_create_webhooks.rb

```diff
@@ -8,7 +8,6 @@ def change
       t.integer :request_method
       t.integer :content_type
       t.boolean :enabled, default: false
-
       t.timestamps null: false
```

Comment:
Nothing changed in this file. You can simply revert the file or create an empty line here to restore the previous state of the file and avoid this change.